### PR TITLE
The scripts should exit if directories are not found

### PR DESCRIPTION
**Pull request by amoghjalan**
Ref: https://github.com/koalaman/shellcheck/wiki/SC2164

## Acceptance Criteria fulfillment

- [ ] Task 1
- [ ] Task 2
- [ ] Task 3

### CODE REVIEW

**Comment by adnanhashmi09** [comment]
Please add an exit code of 127 as well. Otherwise it will exit with the default vode of 0.

File: dev.sh [hunk 1]
```diff
@@ -4,7 +4,7 @@ docker-compose down
 
 cat env.example > .env
 
-cd ./cli
+cd ./cli || exit
```
Comment (adnanhashmi09) [inline dev.sh:7]: Do we want it to exit without error?

Comment (amoghjalan) [reply dev.sh:7]: It doesn't exit without an error earlier it wouldn't have exited at all rather further process would have been tried in the wrong directory. Above shall exit while returning the error from cd

**Comment by amoghjalan** [comment]
> Please add an exit code of 127 as well. Otherwise it will exit with the default code of 0.

127 is used for command not found, rather over here the command is correct the directory may not be available

**Review by e-for-eshaan — APPROVED** [review]

**Pull request merged**